react-instantsearch-hooks/responsive-filtering/src/App.css

```diff
@@ -0,0 +1,199 @@
+/* Global */
+body,
+h1 {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+em {
+  background: cyan;
+  font-style: normal;
+}
+
+#root {
+  overflow: hidden;
+}
+
+/* Header */
+.header {
+  display: flex;
+  align-items: center;
+  min-height: 50px;
+  padding: 0.5rem 1rem;
+  background-image: linear-gradient(to right, #8e43e7, #00aeff);
+  color: #fff;
+  margin-bottom: 1rem;
+}
+
+.header a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.header-title {
+  font-size: 1.2rem;
+  font-weight: normal;
+}
+
+.header-title::after {
+  content: ' ▸ ';
+  padding: 0 0.5rem;
+}
+
+.header-subtitle {
+  font-size: 1.2rem;
+}
+
+.ais-ClearRefinements-button {
+  height: 40px !important;
+}
+
+/* Container */
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+/* Search */
+.search-panel {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 1em;
+}
+
+.search-panel__filters {
+  grid-row: 1 / 3;
+}
+
+.search-panel__pagination {
+  margin: 2rem auto;
+  text-align: center;
+}
+
+.responsive-filters-buttons {
+  display: none;
+  gap: 1em;
+}
+
+.responsive-filters-buttons__sort {
+  width: 100%;
+}
+
+.responsive-filters-modal {
+  width: 100vw;
```

Comment:
TBH I'm still not sold on this pattern. If we want to see the results in the back, I'd rather leave some space at the top of the dialog (like with a native dialog, see picture) because we have more vertical than horizontal space at our disposal.

![image](https://user-images.githubusercontent.com/5370675/203270137-084f6a4d-1b3c-47b3-afb5-6291b7d13216.png)

Reply:
We'll need Katie's help here if we decide to go to that level of detail for the v1 of the guide and its associated guide. I suggest we discuss it again once I've produced the acceptance criteria.